app/views/products/index.html.erb

```diff
@@ -1,12 +1,21 @@
+<% product_class = Product.product_types[@type] %>
+
 <h2 class="apps-section-header">
   <%= products_title(@type) %>
 </h2>
 
 <p>
   <%= link_to 'New Product', new_product_path(type: @type), class: 'btn btn-default' %>
   <span class="pull-right">
-    <%= link_to 'Launch Shell', OodAppkit.shell.url(path: @type == :dev ? DevRouter.base_path : UsrRouter.base_path.realpath).to_s, target: '_blank', class: 'btn btn-default' %>
-    <%= link_to 'Launch Files', OodAppkit.files.url(path: @type == :dev ? DevRouter.base_path : UsrRouter.base_path.realpath).to_s, target: '_blank', class: 'btn btn-default' %>
+    <%= link_to 'Launch Shell', OodAppkit.shell.url(host: ENV['OOD_DEV_SSH_HOST'] || 'default', path: product_class.router.base_path.realpath).to_s, target: '_blank', class: 'btn btn-default' %>
```

Comment:
In several places we have this: `ENV['OOD_DEV_SSH_HOST'] || 'default'`. I wonder if we should require explicitly setting the dev host, since ending up at the `default` would be bad. Does it make more sense to disable the shell buttons altogether or hide the buttons if `OOD_DEV_SSH_HOST` is not set.

When deploying, I could set `OOD_DEV_SSH_HOST=default` if I really wanted to.